### ISSUE TITLE
res_norm=True fails on AstraReconGpu

### ISSUE THREAD

**Issue by elainehoml** [DiamondLightSource/Savu#859]
Hi, 

When running iterative reconstruction with AstraReconGpu, the reconstruction fails when I set res_norm=True. Setting res_norm=False works fine.

The error log for this is found here: 
[log.txt](https://github.com/DiamondLightSource/Savu/files/7337286/log.txt)

Process list to replicate: 
[mrc_AstraCGLS_GPU_resnormFalse.zip](https://github.com/DiamondLightSource/Savu/files/7337306/mrc_AstraCGLS_GPU_resnormFalse.zip)

Test data used was [test_data/data/test_image.mrc](https://github.com/DiamondLightSource/Savu/blob/mrc_test_fixes/test_data/data/test_image.mrc)

Versions:

- savu 4.0
- savu-lite 4.0_2021.10
- mrcfile 1.2.0

Thank you :)

**Comment by dkazanc** [comment]
hi, the bug is now fixed in 2021.11 version. Please try if it works for you. cheers

**Comment by elainehoml** [comment]
Hi @dkazanc,

Thanks for fixing this! I can confirm that this works but in the process I have discovered another bug where the `TiffSaver` plugin doesn't work if `res_norm=True` (sorry :( ).

I've attached process lists here as well.
[process_lists.zip](https://github.com/DiamondLightSource/Savu/files/7845881/process_lists.zip)

The error that I get is  
```
Exception: ERROR: Broken plugin chain. 
 Please name the 1 in_data sets associated with the plugin TiffSaver in the process file.
```

Thank you :)
